docs/get-started/alerting.md

```diff
@@ -6,64 +6,161 @@
 Alerting notifies of important or unusual activity in your database environments so that you can identify and resolve problems quickly. When something needs your attention, PMM automatically sends you an alert through your specified contact points.
 
```

Comment:
Irrespective of all comments that follow (I did this as my last comment) I would make the following recommendations to this work, as a whole:
Topic/chapter order I recommend:
(1) Introduction - what Percona Alerting is, why we use alerting, its value proposition, etc...
(2) General - feature is enabled by default etc...also the difference and or relationships between Alerting Rules, alerts and notifications in PMM 
(3) Alert Templates page
(4) Alert Rules page & Explore page
(5) Fired Alerts (briefly) - I didnt see this page covered at all
(6) Silences Page
(7) Notifications Page & muting
(8) Contacts points page
(9) alert groups and Admin - we should mention them and write something about these, no?

As well, if for no other reason, the Percona Alerting section should be consistent with other PMM documentation, most of which have screenshots. At the very least, there should be a screenshot of the entry page for each topic/page